activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -39,7 +39,6 @@ class RedisCacheStore < Store
         connect_timeout:    20,
```

Comment:
While we're at it, a 20 seconds connect timeout is stupidly massive, and make little sense when combined with a 1 second read timeout. Might as well reduce it too.

Reply:
No, 1 seconds is plenty.